docs/source/en/using-diffusers/ip_adapter.md

```diff
@@ -305,15 +305,32 @@ masks = processor.preprocess([mask1, mask2], height=output_height, width=output_
   </div>
 </div>
 
-When there is more than one input IP-Adapter image, load them as a list to ensure each image is assigned to a different IP-Adapter. Each of the input IP-Adapter images here correspond to the masks generated above.
+When there is more than one input IP-Adapter image, you have two methods to assign images to IP Adapters: one is to load them as a list to ensure that each image is assigned to a different IP-Adapter, the other is assign them all to the same IP Adapter. Each of the input IP-Adapter images here correspond to the masks generated above.
```

Comment:
umm, we should always assign a list of images to the same IP Adapter instead of loading the same adapter multiple times: even though the latter would work, we don't recommend it so no need to mention it on the doc IMO

Reply:
I agree, now updated